File: challenge-questions/lambdapark/ans.cpp
#include <stdio.h>
using namespace std;

int main() {
    int children;
    scanf("%d", &children);
    int adults;
    scanf("%d", &adults);
    int discount;
    if (children + adults >= 10) {
        discount = 10;
        if (children >= 5) {
            discount = 20;
        }
    }

    int total = 10000 * adults + 8000 * children;
    total = total * (100 - discount) / discount;
    printf("%d", total);
    return 0;
}

File: challenge-questions/lambdapark/ans.py
children = int(input())
adults = int(input())

total = 10000 * adults + 8000 * children
discount = 0
if (children + adults >= 10):
  discount = 10
  if (children >= 5):
    discount = 20
total *= (100 - discount) / 100
print(int(total))
